shared/AppInsightsCore/Tests/Unit/src/CookieManager.Tests.ts

```diff
@@ -1223,10 +1223,13 @@ export class CookieManagerTests extends AITestClass {
                 this.clock.tick(1);
 
                 // Cookie should be flushed with the maxAge parameter
-                let expectedValue = this.isEmulatingIe ? 
-                    newValue + "; path=/; expires=" : 
-                    newValue + "; path=/; expires=";
-                Assert.ok(this._testCookies[newKey].indexOf(expectedValue) === 0, "Cookie should be flushed with expires");
+                let cookieValue = this._testCookies[newKey];
+                Assert.ok(cookieValue, "Cookie should be flushed to storage");
+                Assert.ok(cookieValue.indexOf(newValue + "; path=/; expires=") === 0, "Cookie should start with value, path and expires");
```

Comment:
The order of the path and expires is still wrong, as the runtime is reordering the properties based on how it's populating the object in _formatDeletionValue.
Update so that the order doesn't matter